### PR TITLE
Updated README.md Added Precise Swift Description

### PR DESCRIPTION
**Pull request by calijoefornium**
Added Precise Swift Description

### CODE REVIEW

File: README.md
Comment (buttaface) [inline README.md:44]: This reads like an ad, is it really needed?

Comment (yashhchitroda) [reply README.md:44]: Hey @buttaface Thanks for reviewing I would like keep my point for your Query Asked 

No it dosen't its just like providing some feature of Swift that is needed incase to understand advantages of swift over other Languages

**Comment by yashhchitroda** [comment]
@swift-ci please smoke test and merge

**Comment by yashhchitroda** [comment]
@swift-ci please test

**Comment by yashhchitroda** [comment]
> Added Precise Swift Description

@swift-ci Please test macOS

**Comment by buttaface** [comment]
Unless you have permission from the Swift project, you will not be able to test or merge.

@varungandhi-apple, perhaps you'd like to chime in.

**Comment by yashhchitroda** [comment]
> Unless you have permission from the Swift project, you will not be able to test or merge.
> 
> @varungandhi-apple, perhaps you'd like to chime in.

@buttaface  so what's the step should I need to perform in case to merge.

**Comment by buttaface** [comment]
You need someone with commit access to review, approve, and merge for you: take a look at [the doc for how it works](https://github.com/apple/swift/blob/main/docs/HowToGuides/FirstPullRequest.md).

**Comment by yashhchitroda** [comment]
> You need someone with commit access to review, approve, and merge for you: take a look at [the doc for how it works](https://github.com/apple/swift/blob/main/docs/HowToGuides/FirstPullRequest.md).

can you mention or know someone who has commit access to review, approve, and merge access so it would help me as well

**Comment by buttaface** [comment]
@benrimmington, you're one of the last people to modify this file, what do you think?

**Comment by yashhchitroda** [comment]
> @benrimmington, you're one of the last people to modify this file, what do you think?

@benrimmington can you help me out if possible

**Comment by yashhchitroda** [comment]
> @benrimmington, you're one of the last people to modify this file, what do you think?

@buttaface I think  @benrimmington isn’t available or maybe inactive so @buttaface can you approve my pull request if possible

**Comment by yashhchitroda** [comment]
@DougGregor can you merge my Pull Request ??

**Review by ejcheng — CHANGES_REQUESTED** [review]
> This reads like an ad, is it really needed?

@buttaface I would say you are right, the README seems to be a cozy way to welcome new users of Swift, and it is not supposed to be the beginning of a tutorial. The additions in this PR don't really fit well with the existing tone of the document.

**Comment by yashhchitroda** [comment]
> > This reads like an ad, is it really needed?
> 
> @buttaface I would say you are right, the README seems to be a cozy way to welcome new users of Swift, and it is not supposed to be the beginning of a tutorial.

@import-brain thanks for reviewing I can change that but is my rest of content relatable??

**Comment by import-brain** [comment]
> > > This reads like an ad, is it really needed?
> > 
> > 
> > @buttaface I would say you are right, the README seems to be a cozy way to welcome new users of Swift, and it is not supposed to be the beginning of a tutorial.
> 
> @import-brain thanks for reviewing I can change that but is my rest of content relatable??

The content is great, I just personally don't believe that README would be the right place for it. Perhaps there's a tutorial doc somewhere in the repo? I can look further...

**Comment by yashhchitroda** [comment]
> @buttaface I would say you are right, the README seems to be a cozy way to welcome new users of Swift, and it is not supposed to be the beginning of a tutorial. The additions in this PR don't really fit well with the existing tone of the document.

@import-brain I have changed the heading if my rest of contents are good enough then can you approve my PR if possible ??

**Review by ejcheng — APPROVED** [review]

**Comment by import-brain** [comment]
> > @buttaface I would say you are right, the README seems to be a cozy way to welcome new users of Swift, and it is not supposed to be the beginning of a tutorial. The additions in this PR don't really fit well with the existing tone of the document.
> 
> @import-brain I have changed the heading if my rest of contents are good enough then can you approve my PR if possible ??

@yashhchitroda approved, I'm not a maintainer, so I can't merge

**Comment by yashhchitroda** [comment]
@import-brain Thanks for approving Can you guide me how can I merge my PR if possible

**Comment by import-brain** [comment]
> @import-brain Thanks for approving Can you guide me how can I merge my PR if possible

Sure! We do not have permissions to merge PRs in this repository, but we can ask other people who have permissions to do it, e.g. tbkka (not @'ing them because they get notifications every time they are mentioned). I'm not sure who the write access people are besides tbkka, but if you find out, you could mention (@) them here.

If one of the people who has merge access decides your PR is good, then they will merge it for you.

**Comment by yashhchitroda** [comment]
> > @import-brain Thanks for approving Can you guide me how can I merge my PR if possible
> 
> Sure! We do not have permissions to merge PRs in this repository, but we can ask other people who have permissions to do it, e.g. tbkka (not @'ing them because they get notifications every time they are mentioned). I'm not sure who the write access people are besides tbkka, but if you find out, you could mention (@) them here.
> 
> If one of the people who has merge access decides your PR is good, then they will merge it for you.

can you mention them on behalf of mysids if possible because I am finding a bit difficult

**Comment by yashhchitroda** [comment]
# Please Merge
@tbkka @DougGregor @slavapestov @jckarter @gottesmm
@lattner @jrose-apple @gribozavr @xedin @rjmccall
@practicalswift @aschwaighofer @eeckstein @compnerd 
@nkcsgexi @rudkx @rintaro

**Comment by yashhchitroda** [comment]
> # Please Merge
> @tbkka @DougGregor @slavapestov @jckarter @gottesmm @lattner @jrose-apple @gribozavr @xedin @rjmccall @practicalswift @aschwaighofer @eeckstein @compnerd @nkcsgexi @rudkx @rintaro

Please merge anyone

**Comment by yashhchitroda** [comment]
# Please Merge to main 
@tbkka @DougGregor @slavapestov @jckarter @gottesmm
@lattner @jrose-apple @gribozavr @xedin @rjmccall
@practicalswift @aschwaighofer @eeckstein @compnerd 
@nkcsgexi @rudkx @rintaro

**Comment by buttaface** [comment]
@yashhchitroda, mentioning a bunch of unrelated names like that is not going to work and is more likely to get you added to their ignore lists.

What I did was check the commit history for this file, and then saw which of the recent authors showed recent activity on their github profile before pinging one person, Ben. He didn't respond, but it is now the weekend and this project is fairly quiet on weekends.

Also, marketing fluff like this in the doc is not a high priority, so obviously committers will put off reviewing pulls like this while they work on actual bug fixes or features. You might be waiting for weeks or months, and continually pinging people in the meantime is not going to endear you to them, ie you will make it take longer.

**Comment by yashhchitroda** [comment]
> @yashhchitroda, mentioning a bunch of unrelated names like that is not going to work and is more likely to get you added to their ignore lists.
> 
> What I did was check the commit history for this file, and then saw which of the recent authors showed recent activity on their github profile before pinging one person, Ben. He didn't respond, but it is now the weekend and this project is fairly quiet on weekends.
> 
> Also, marketing fluff like this in the doc is not a high priority, so obviously committers will put off reviewing pulls like this while they work on actual bug fixes or features. You might be waiting for weeks or months, and continually pinging people in the meantime is not going to endear you to them, ie you will make it take longer.

@buttaface First of all Thanks for Reaching me I highly appreciate your dedication and loyalty towards me as I am new to GitHub’s PR stuffs I wasn’t getting things merged so I was in an tense situation that might be due to some mistakes my comments won’t be sent to people who has Write access so I was continuously trying to keep them updated with respect to pull request but as you explain things now I am clear about the process 

                                          -Thanks for reaching out

File: README.md
Comment (WowbaggersLiquidLunch) [inline README.md:42]: I think this line makes the description _less_ precise, and furthers this misconception that it's a Apple/Darwin-only language.

The rest of the additions are already covered on the front page of swift.org.

File: README.md
Comment (WowbaggersLiquidLunch) [inline README.md:28]: The current capitalisation is correct. Propositions (unless at the start of sentence) are not capitalised in English titles: https://en.wikipedia.org/wiki/Title_case

File: README.md
Comment (yashhchitroda) [reply README.md:42]: @WowbaggersLiquidLunch thanks for reaching as you mentioned your point I support your point

 but I don't think so it's less precise form anypoint of view, regarding swift.org well if it's in README it won't affect them either

File: README.md
Comment (yashhchitroda) [reply README.md:28]: Heading are always capatalized in order to grab users attraction

File: README.md
Comment (import-brain) [reply README.md:42]: > I think this line makes the description _less_ precise, and furthers this misconception that it's a Apple/Darwin-only language.
> 
> 
> 
> The rest of the additions are already covered on the front page of swift.org.

@WowbaggersLiquidLunch I concur with this, Swift can be used for a lot more than developing apps for Apple products.

Comment (ktoso) [reply README.md:42]: I agree this isn't very good wording, is a great _general purpose_ language and there is no need to highlight apple platforms at the exclusion of others like this.

**Comment by ktoso** [comment]
Thanks for the effort but personally I don't feel this is really helping the readme much to be honest 🤔 
It isn't clear how this would make it more compelling and welcoming to new and different users of the language.

**Comment by yashhchitroda** [comment]
> Thanks for the effort but personally I don't feel this is really helping the readme much to be honest thinking It isn't clear how this would make it more compelling and welcoming to new and different users of the language.

@ktoso  Thanks for reaching me I don't think so maybe it will be helpful to some extent but anyways choice is up to you if you feel you can merge else close PR !!

**Comment by ktoso** [comment]
Thanks again @yashhchitroda, but this specific PR we don't feel is improving the readme's feel, so I'll close the PR.

**Pull request closed without merging**
